quickstart/sources.go

```diff
@@ -0,0 +1,172 @@
+package quickstart
+
+import (
+	"fmt"
+	"net/url"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/huh"
+	"github.com/pkg/errors"
+	"github.com/speakeasy-api/sdk-gen-config/workflow"
+	"github.com/speakeasy-api/speakeasy/charm"
+)
+
+func sourceBaseForm(inputWorkflow *workflow.Workflow) (*State, error) {
+	source := &workflow.Source{}
+	var sourceName string
+	if _, err := tea.NewProgram(charm.NewForm(huh.NewForm(
+		huh.NewGroup(
+			huh.NewInput().
+				Title("A name for this source:").
```

Comment:
So I ended up instantly making a mistake here. I accidentally input `./openapi.yaml`. I think it's probably confusing to a new user to have to "name a source" when they don't even know what a source is. Also, the helper text `unique name across this workflow` isn't helpful either because what is a workflow?

I think we should simplify this whole setup as much as possible. So for example I'd just automatically name this `my-first-source` or something like that. No need to prompt the user at all.

Reply:
I think we can fill in a default but I'm not sure we should just totally obfuscate it from the user